lib/facter.rb.in

```diff
@@ -27,8 +27,9 @@ module Facter
   else
     # Simply require libfacter.so; this will define all of the Facter API
     begin
-      facter_dir = File.join(File.expand_path("#{File.dirname(__FILE__)}"), '${LIBFACTER_INSTALL_RELATIVE}')
-      require "#{ENV['FACTERDIR'] || facter_dir}/${LIBFACTER_INSTALL_DESTINATION}/libfacter.so"
+      facter_dir = ENV['FACTERDIR'] || File.join(File.expand_path("#{File.dirname(__FILE__)}"), '${LIBFACTER_INSTALL_RELATIVE}')
+      ENV['PATH'] = "#{File.join(facter_dir, 'bin')}#{File::PATH_SEPARATOR}#{ENV['PATH']}"
```

Comment:
It was before. I suppose on non-Windows, the prior behavior was not to have things on the path, so I guess it should still be blocked. I'll update this.